tests/test_homepage.py

```diff
@@ -5,5 +5,4 @@ def test_home_page(test_client):
     THEN check the response is valid
     """
     response = test_client.get('/')
-    assert response.status_code == 200
-    assert b"MusicXML to ABC Notation Converter" in response.data
```

Comment:
Just curious why this line has been removed?

Reply:
ops, that was a mistake.